pkg/services/store/entity/sqlstash/sql_storage_server_test.go

```diff
@@ -150,3 +48,18 @@ func setUpTestServer(t *testing.T) entity.EntityStoreServer {
 	require.NoError(t, err)
 	return s
 }
+
+// TODO: remove all the following once the Proposal 1 for Consistent Resource
+// Version is finished.
```

Comment:
What is this for ?

Reply:
Silences the `unused` linter. These items will be used in followup PRs, and these items are witnesses that they all must be used or otherwise removed.